_studio/mfx_lib/decode/vp9/src/mfx_vp9_dec_decode_hw.cpp

```diff
@@ -945,6 +945,50 @@ mfxStatus VideoDECODEVP9_HW::PrepareInternalSurface(UMC::FrameMemID &mid, mfxFra
     return MFX_ERR_NONE;
 }
 
+
+static mfxStatus CheckFrameInfo(mfxFrameInfo &currInfo, mfxFrameInfo &info)
+{
+    mfxStatus sts = CheckFrameInfoCommon(&info, MFX_CODEC_VP9);
+    MFX_CHECK(sts == MFX_ERR_NONE, sts);
```

Comment:
```
MFX_CHECK_STS(sts);
```

Reply:
MFX_CHECK_STS has 2 arguments, second is return code. I think, we need return actual return code from CheckFrameInfoCommon here, not other constant.